src/gobexport/exporter/config/brk.py

```diff
@@ -1045,9 +1065,9 @@ def if_vot_relation(self, trueval: str, falseval: str):
     def get_format(self):
         return {
             'BRK_KOT_ID': 'identificatie',
-            'KOT_AKRKADGEMEENTECODE_CODE': 'aangeduidDoorKadastralegemeentecode.broninfo.code',
-            'KOT_AKRKADGEMEENTECODE_OMS': 'aangeduidDoorKadastralegemeentecode.broninfo.omschrijving',
-            'KOT_SECTIE': 'aangeduidDoorKadastralesectie.[0].bronwaarde',
+            'KOT_AKRKADGEMEENTECODE_CODE': 'aangeduidDoorKadastralegemeentecode.[0].broninfo.broninfo.code',
+            'KOT_AKRKADGEMEENTECODE_OMS': 'aangeduidDoorKadastralegemeentecode.[0].broninfo.broninfo.omschrijving',
```

Comment:
This can't work in the streaming GraphQL API, as this is a bug in the 'normal' GraphQL API (in which case you should fix the API, rather than work around it..)

Reply:
fixed